### PR TITLE
Change hover over message of Column

### PR DESCRIPTION
**Pull request by orzyyyy**
#13312

### CODE REVIEW

File: components/table/demo/head.md [hunk 1]
```diff
@@ -27,7 +28,13 @@ If a `sortOrder` or `defaultSortOrder` is specified with the value `ascend` or `
...
 
 const columns = [{
-  title: 'Name',
+  title: (
+    <div title="title1">
+      <div title="title2">
+        <div title="title3">Name</div>
+      </div>
+    </div>
+  ),
```
Comment (afc163) [inline components/table/demo/head.md]: No need to change the demo.

File: components/table/Table.tsx [hunk 2]
```diff
@@ -855,6 +856,20 @@ export default class Table<T> extends React.Component<TableProps<T>, TableState<
     return title;
   }
 
+  getColumnTitle: any = (node: any, parentNode: any) => {
+    if(!node) {
+      return;
+    }
+    if (typeof node !== 'string') {
```
Comment (ztplz) [inline components/table/Table.tsx]: https://github.com/ant-design/ant-design/blob/master/components/table/interface.tsx#L13
It maybe a `Number` or `Function`.

**Review by ztplz — APPROVED** [review]

File: components/table/Table.tsx (hunk 2 above)
Comment (zy410419243) [reply components/table/Table.tsx]: done

File: components/table/demo/head.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ order: 7
 title:
   en-US: Filter and sorter
   zh-CN: 筛选和排序
+  debug: true
```
Comment (yutingzhao1991) [inline components/table/demo/head.md]: remove.

**Comment by yutingzhao1991** [comment]
这个貌似没有 squash，加了好多 commit 记录。

**Pull request merged**